### DISCUSSION TITLE
I can't install from microsoft store

### DISCUSSION THREAD

**Discussion by fakirpic** [File-New-Project/EarTrumpet#724] [General]
microsoft store doesn't work for me, that is the error code 0x80070422.
is there any other way to install this app?

**Comment by riverar** [comment]
Are your Background Intelligent Transfer Service and Windows Update services disabled by any chance?

**Comment by fakirpic** [comment]
nope they are running, i just checked many instructions. but still i can't install any app from store.

**Comment by riverar** [comment]
@fakirpic Do you have any other disabled services?

**Comment by fakirpic** [comment]
@riverar no mate,i tried many instructions like that 
https://answers.microsoft.com/en-us/windows/forum/windows_8-windows_install/i-cant-download-apps-from-windows-store-error-code/b14d144a-5432-46b9-ba01-f1228abf4ca1
none of them worked for me
i couldn't find any built here, just installed from here https://github.com/Nonononoki/EarTrumpet/releases

**Comment by Originalme** [comment]
@fakirpic can you install any other applications from the store?

**Comment by fakirpic** [comment]
> @fakirpic can you install any other applications from the store?

nope also now microsoft store closes itself immediately on start :) , i don't know how to fix someone else's coded software no any information releated the problem

![image](https://user-images.githubusercontent.com/5578194/58738133-30391780-840d-11e9-982f-a3724da0f34a.png)

**Comment by riverar** [comment]
@fakirpic Doesn't sound good. Have you tried running `WSReset.exe`, waiting a minute, then trying again?

**Comment by fakirpic** [comment]
> @fakirpic Doesn't sound good. Have you tried running `WSReset.exe`, waiting a minute, then trying again?

yes , it also just opens and closes microsoft store immediately :)

**Comment by GoldenTao** [comment]
Can you see if another user on your PC has the same issues with the store? If so, it's likely your Windows install, if not, maybe your profile has an issue.

**Comment by scatterkeir** [comment]
I just had this issue, and nothing seemed to fix it, I just gave up - then several hours later I got a notification in the action center that it had installed, and it was fine.

**Comment by riverar** [comment]
Can you navigate to Event Viewer > Applications and Services Logs > then check each log indicated below for any errors? (Be sure to turn on Analytic and Debug logs via the View menu.)

• Microsoft > Windows >  Bits-Client > Operational
• Microsoft > Windows > AppReadiness > Admin / Debug / Operational
• Microsoft > Windows > AppxDeployment > Operational
• Microsoft > Windows > Store > Operational
• Microsoft > Windows > AppX[xxxx] [each one] > Operational
• Microsoft > Windows > WindowsUpdateClient > Operational

Can you also attach `%WinDir%\softwaredistribution\reportingevents.log`?

**Comment by Luckz** [comment]
Are you on 1903? After I force-upgraded to that my Store came back to life.

**Comment by scatterkeir** [comment]
I'm on 1903, but I was already on 1903 when I had the problem in the first place.

Stuff that seems like it might be relevant in the logs (I'm just a punter who doesn't really know about this sort of stuff at all, so don't attach too much weight to my guesses):

**• Microsoft > Windows > Store > Operational**

Event 1, Store-SDK
Failure invoking ms-windows-store://pdp/?productId=9NBLGGH516XP&ocid=&cid=&referrer=unistoreweb&scenario=click&webig=09b83bcd-ef51-4188-918e-e2059ffe85b6&muid=13ABCEDBB85D651809ACC22AB9B7641B&websession=&tduid=: action pdp not recognized
Error Code: Unknown HResult Error code: 0x80000000
Function: InvokeUri
Source: C:\BA\43\s\src\Core\ProtocolActions\ProtocolActionHandler.cs (108)

Event 1, Store-SDK
ProductId: 9NBLGGH516XP
Error Code: Unknown HResult Error code: 0x80244007
Function: ItemStatusChanged
Source: C:\BA\43\s\src\App\Acquisition\DefaultAppInstallManager.cs (1107)

several of the above one, presumably because I kept trying clicking install.

something possibly related - it looked like the licence manager was having trouble renewing its lease or something like that - timer expiring - there don't seem to have been any of those error messages since the install successfully completed. Is it possible that after a couple of hours EarTrumpet was able to install successfully on my PC because the license manager stopped having problems?

• Microsoft > Windows > WindowsUpdateClient > Operational

this has a failure to check for updates with error 0x80244007 and a subsequent successful check a few hours later - times in the ballpark of the EarTrumpet installation problem and resolution - perhaps this suggests a general BITS outage causing both?

attaching the log you requested, I assum that the AGENT_DETECTION_FAILED this afternoon, which is in the ballpark of the time of the problems, is pertinent.
[ReportingEvents.log](https://github.com/File-New-Project/EarTrumpet/files/3463058/ReportingEvents.log)

**Comment by fakirpic** [comment]
well also microsoft edge Error Code: INET_E_RESOURCE_NOT_FOUND
![image](https://user-images.githubusercontent.com/5578194/66449390-938f2200-ea5d-11e9-9999-585257ccf942.png)

windows update 0x80070422
![image](https://user-images.githubusercontent.com/5578194/66449398-9d188a00-ea5d-11e9-85ed-6eb78811db04.png)

and other microsoft things don't work too,not sure what i did, i tried to enable almost all the services, changed to my dns to 8.8.8.8 instad of doh not worked again.

**Comment by dyennam** [comment]
Hello @riverar I got a similar issue, and here are my ERROR logs for the issue. I reset windows store and tried other methods aswell. Any Help would be appreciated.
"AppXDeployment-Server -> Microsoft-Windows-AppXDeploymentServer/Operational"

- AppX Deployment operation failed for package 40459File-New-Project.EarTrumpet_2.1.5.0_neutral_~_1sdd7yawvg6ne with error 0x80073CF6. The specific error text for this failure is: An internal error occurred with error 0x80073D05. See http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app deployment issues.
- Deployment Register operation with target volume C: on Package 40459File-New-Project.EarTrumpet_2.1.5.0_neutral_~_1sdd7yawvg6ne from:  (AppxBundleManifest.xml)  failed with error 0x80073CF6. See http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app deployment issues.
- Finished servicing package 40459File-New-Project.EarTrumpet_2.1.5.0_x86__1sdd7yawvg6ne. Setting the package state to enabled returned with  0x0.
- error 0x8007139F: While removing the request, the system failed to de-register the windows.MRT extension due to the following error: The group or resource is not in the correct state to perform the requested operation.

**Reply by HppZ** [reply]
https://github.com/MicrosoftDocs/msix-docs/issues/271

**Reply by HppZ** [reply]
no solution?

**Comment by Jo2Mayer** [comment]
> microsoft store doesn't work for me, that is the error code 0x80070422. is there any other way to install this app?

Try to fix it by heading into C:\Users\<USERNAME>\AppData\Local\Packages\Microsoft.Windows.Search_cw5n1h2txyewy\LocalState\AppIconCache\100
and delete
40459File-New-Project_EarTrumpet_1sdd7yawvg6ne!EarTrumpet

I hope it will help you
